sentry-ruby/spec/sentry/utils/real_ip_spec.rb

```diff
@@ -50,6 +50,19 @@
     end
   end
 
+  context "when custom proxies are provided" do
+    subject do
+      Sentry::Utils::RealIp.new(
+        :forwarded_for => "2.2.2.2, 3.3.3.3, 4.4.4.4",
+        :trusted_proxies => ["4.4.4.4"]
+      )
+    end
+
+    it "should return REMOTE_ADDR" do
```

Comment:
this description should be updated I think?

Reply:
Yes, sir!